packages/flutter_tools/test/general.shard/windows/windows_workflow_test.dart

```diff
@@ -5,22 +5,19 @@
 import 'package:flutter_tools/src/base/platform.dart';
 import 'package:flutter_tools/src/features.dart';
 import 'package:flutter_tools/src/windows/windows_workflow.dart';
-import 'package:mockito/mockito.dart';
 
 import '../../src/common.dart';
 import '../../src/context.dart';
 import '../../src/testbed.dart';
 
 void main() {
   Testbed testbed;
-  MockPlatform windows;
-  MockPlatform notWindows;
+  FakePlatform windows;
+  FakePlatform notWindows;
```

Comment:
This one wasn't `MockFile` related but it was easy.

Reply:
I'll take it